inc/opcodes.h

```diff
@@ -20,28 +20,268 @@
 /*									*/
 /************************************************************************/
 
+enum opcodes {
+    opc_unused_0 = 0,
+    opc_CAR = 1,
+    opc_CDR = 2,
+    opc_LISP = 3,
+    opc_NTYPX = 4,
+    opc_TYPEP = 5,
+    opc_DTEST = 6,
+    opc_UNWIND = 7,
+    opc_FN0 = 8,
+    opc_FN1 = 9,
+    opc_FN2 = 10,
+    opc_FN3 = 11,
+    opc_FN4 = 12,
+    opc_FNX = 13,
+    opc_APPLYFN = 14,
+    opc_CHECKAPPLY = 15,
+    opc_RETURN = 16,
+    opc_BIND = 17,
+    opc_UNBIND = 18,
+    opc_DUNBIND = 19,
+    opc_RPLPTR_N = 20,
+    opc_GCREF = 21,
+    opc_ASSOC = 22,
+    opc_GVAR_ = 23,
+    opc_RPLACA = 24,
+    opc_RPLACD = 25,
+    opc_CONS = 26,
+    opc_CMLASSOC = 27,
+    opc_FMEMB = 28,
+    opc_CMLMEMBER = 29,
+    opc_FINDKEY = 30,
+    opc_CREATECELL = 31,
+    opc_BIN = 32,
+    opc_BOUT = 33,
+    opc_POPDISP = 34,
+    opc_RESTLIST = 35,
+    opc_MISCN = 36,
+    opc_unused_37 = 37,
+    opc_RPLCONS = 38,
+    opc_LISTGET = 39,
+    opc_unused_40 = 40,
+    opc_unused_41 = 41,
+    opc_unused_42 = 42,
+    opc_unused_43 = 43,
+    opc_EVAL = 44,
+    opc_ENVCALL = 45,
+    opc_TYPECHECK = 46,
+    opc_STKSCAN = 47,
+    opc_BUSBLT = 48,
+    opc_MISC8 = 49,
+    opc_UBFLOAT3 = 50,
+    opc_TYPEMASK_N = 51,
+    opc_RDPROLOGPTR = 52,
+    opc_RDPROLOGTAG = 53,
+    opc_WRTPTRTAG = 54,
+    opc_WRTPTR0TAG = 55,
+    opc_MISC7 = 56,
+    opc_DOVEMISC = 57,
+    opc_EQL = 58,
+    opc_DRAWLINE = 59,
+    opc_STORE_N = 60,
+    opc_COPY_N = 61,
+    opc_RAID = 62,
+    opc_SLRETURN = 63,
+    opc_IVAR1 = 64,
```

Comment:
I think the convention for the variable slots is that they're numbered from 0.  The reference being looking at the results of (INSPECTCODE 'fn-name)

Reply:
Updated for that.